File: 1-easy/problems/2455-average-value-of-even-numbers-that-are-divisible-by-three/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def averageValue(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        total = cnt = 0
        for x in nums:
            if x%6:
                continue
            total += x
            cnt += 1
        return total//cnt if cnt else 0

File: 1-easy/problems/2455-average-value-of-even-numbers-that-are-divisible-by-three/solution.ts
function averageValue(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  total = cnt = 0
  for (x in nums) {
      if (x%6) {
          continue
      total += x
      cnt += 1
  return total//cnt if cnt else 0;
}

export { averageValue };
